interface/controller/app_controller.go

```diff
@@ -0,0 +1,6 @@
+package controller
+
+// AppController struct
+type AppController struct {
+	Pokemon interface{ PokemonController }
```

Comment:
Here I define `AppController` will have for the moment only one controller, `PokemonController`.